x-pack/plugins/fleet/server/types/rest_spec/message_signing_service.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RotateKeyPairSchema } from './message_signing_service';
+
+describe('RotateKeyPairSchema', () => {
+  it('should allow without any query', () => {
+    expect(() => RotateKeyPairSchema.query.validate({})).not.toThrow();
```

Comment:
Ok. So I don't have background on this API, but this tests seems to suggest that you DO want to do the validation on the handler and not leave it to the schema?

Reply:
we decided to move the validation to the schema d895980d1ce97a1fddbbd60817d3ec9cfd89e2a6